### ISSUE TITLE
您好! 编译make不行呀

### ISSUE THREAD

**Issue by chenbinCBcau** [dongdongdong1217/yolov5-deepstream-test3#1]

**Comment by NeuroDong** [comment]
首先，请保证你是jetson上进行编译的，然后请保证你的jetson安装了deepstream，其次，要把这个这个项目和deepstream里面的test例子项目放在同一个文件下，然后进行编译。如果还有问题，可以把你的报错截图发出来。

**Comment by chenbinCBcau** [comment]
对 我就是在jetson上下载好了deepstream5.1   将该文件复制到了/opt/nvidia/deepstream/deepstream-5.1/sources/apps/sample_apps下面 make
make:Nothing to be done for "all";
而且up的执行命令LD_PRELOAD=./libmyplugins.so ./deepstream-test3-app file:///home/ubuntu/video1.mp4
在我这边也不太对 要用LD_PRELOAD=./libmyplugins.so  deepstream-test3-app  file:///home/ubuntu/video1.mp4
是不是deepstream版本的问题

**Comment by NeuroDong** [comment]
make:Nothing to be done for "all"，这个应该不是错，它告诉你，不需要编译了，因为我里面已经有编译好的文件了，你没进行更改的话，是不需要重新进行编译的。

**Comment by NeuroDong** [comment]
你试过我那个执行命令嘛，应该就是我那个呀，在我的jetson上都要加一个./的。

**Comment by chenbinCBcau** [comment]
![Screenshot from 2021-07-06 00-24-50](https://user-images.githubusercontent.com/73467065/124572225-10da0780-de7b-11eb-935e-1e1031875b11.png)
up看下

**Comment by NeuroDong** [comment]
配置文件路径的问题，仔细检查一下教程里面的step3的操作

**Comment by chenbinCBcau** [comment]
对了 up主做过在nano上用usb摄像头实时检测识别和计数吗？up加下你联系方式吧 这上面沟通不太方便 可以吗

**Comment by NeuroDong** [comment]
我微信号：dongjinzong1217

**Comment by chenbinCBcau** [comment]
对了 up主做过在nano上用usb摄像头实时检测识别和计数吗

**Comment by chenbinCBcau** [comment]
好的 谢谢
